### PR TITLE
fix: extensions list into `settings` field of `import` rules

### PR DESCRIPTION
**Pull request by wakamsha**
## What changed / motivation ?

Fixed each extensions list in the `settings` field of the `import` rule. The main changes are as follows:

- added a missing `.`
  ```diff
  - extensions: ['ts', 'mts'],
  + extensions: ['.ts', '.mts'], 
  ```
- added TypeScript-related extensions

These settings are not necessary when using `typescript-eslint` in conjunction with `essentials`, but they are required when using `essentials` alone, so I have fixed them.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Chore (routine task, maintenance, or non-functional change that doesn't modify src or test files)

## Pre-flight checklist

- [x] I have read the [Contributing Guidelines](https://github.com/moneyforward/eslint-config-moneyforward/blob/main/.github/CONTRIBUTING.md)
- [x] Performed a self-review of my code

## References

- https://github.com/import-js/eslint-plugin-import?tab=readme-ov-file#importextensions

### CODE REVIEW

File: rules/imports.js [hunk 1]
```diff
@@ -18,13 +18,13 @@ module.exports = {
   settings: {
     'import/resolver': {
       node: {
-        extensions: ['.js', '.mjs', '.json', 'ts', 'mts'],
+        extensions: ['.js', '.mjs', '.json', '.ts', '.mts'],
```
Comment (tongari07) [inline rules/imports.js:21]: Is it not necessary to add '.tsx' here?
If not, please state the reason in the code comments or in the PR description.

**Review by tongari07 — APPROVED** [review]
Thanks! LGTM 👍 
I commented on one thing. Please correct if necessary. If you don't need it, you can merge this PR.

File: rules/imports.js (hunk 1 above)
Comment (wakamsha) [reply rules/imports.js]: Considering RSC it would be appropriate to add them as well, so I'd added them.

File: rules/imports.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ module.exports = {
     // TODO: Remove this once eslint-plugin-import supports Flat Config completely.
     // https://github.com/import-js/eslint-plugin-import/issues/2556#issuecomment-1419518561
     'import/parsers': {
-      espree: ['.js', '.mjs', '.jsx', 'ts', 'mts', 'tsx'],
+      espree: ['.js', 'cjs', '.mjs', '.jsx'],
```
Comment (wakamsha) [inline rules/imports.js:36]: Since it is unnecessary, extensions related TypeScript were removed.

**Pull request merged**